src/pybind/mgr/cephadm/module.py

```diff
@@ -3296,6 +3296,8 @@ def _create_crash(self, daemon_id, host):
             'caps': ['mon', 'profile crash',
                      'mgr', 'profile crash'],
         })
+        if ret:
+            raise OrchestratorError(f'Failed to create crash keyring: {ret} "{err}"')
```

Comment:
There are many instances where the return code is not checked...

Should we create a wrapper that throws during failure: `mon_command_throws(...)` ?

Reply:
https://github.com/ceph/ceph/blob/9a3aecbaf3c42b48ba1dc1fdf670c3da01c14956/src/pybind/mgr/dashboard/services/ceph_service.py#L147-L183

Reply:
and actually, I really miss 

https://github.com/sebastian-philipp/ceph/blob/6fb514079204fed76e805b9a863704c523716598/src/python-common/ceph/mon_command_api/generated_api.py 

But I never managed to get something like this into the tree yet. I think @matthewoliver would also like to have it.